setup.py

```diff
@@ -12,8 +12,10 @@
 from typing import List
 
 from setuptools import find_packages, setup
-from torchsnapshot import __version__
 
+# using exec_file instead of the import to avoid having to install dependencies
+# when building the wheel
+exec(open('torchsnapshot/version.py').read())
```

Comment:
```suggestion
# using exec_file instead of the import to avoid having to install dependencies
# when building the wheel
exec(open('torchsnapshot/version.py').read())

```
for pre-commit